bundles/GraphicServiceOrder/Controller/GraphicServiceOrderController.php

```diff
@@ -46,12 +46,13 @@ public function createOrder(Request $request, OrderService $orderService)
             $orderService->createOrder($gsOrder, $uploadedFiles);
 
             // Go to form submitted page.
-            return $this->redirectToRoute('graphic_service_order_submitted');
+            return $this->redirectToRoute('graphic_service_order_submitted', [$gsOrder->getId()]);
```

Comment:
Unless I'm learning something new here, I suppose this should be 
```
$this->redirectToRoute('graphic_service_order_submitted', [$gsOrder->getId()])
```

Reply:
I'm unsure what's different?

Reply:
Nothing. `'id' => ` was missing – it has now been added. See comment on `createOrderSubmitted`.

Reply:
fixed